kolibri/plugins/registry.py

```diff
@@ -49,8 +49,8 @@ def initialize():
                 plugin_module = importlib.import_module(app + ".kolibri_plugin")
                 logger.debug("Loaded kolibri plugin: {}".format(app))
                 plugin_classes = []
-                for obj in plugin_module.__dict__.values():
-                    if type(obj) == type and obj is not KolibriPluginBase and issubclass(obj, KolibriPluginBase):
+                for obj in plugin_module.PLUGINS:
```

Comment:
:+1: My intuition was that __all__ would involve more module introspection, so this naively seemed cleaner. Glad it is also cleaner too!